codes.merritt.FeelingFinder.yml

```diff
@@ -0,0 +1,31 @@
+---
+app-id: codes.merritt.FeelingFinder
+runtime: org.gnome.Platform
+runtime-version: "41"
+sdk: org.gnome.Sdk
+command: feeling_finder
+separate-locales: false
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--device=dri"
+  - "--socket=pulseaudio"
+  - "--share=network"
+modules:
+  - name: feeling_finder
+    buildsystem: simple
+    build-commands:
+      - "./build-flatpak.sh"
+    sources:
+      - type: file
+        path: FeelingFinder-Linux-Portable.tar.gz
```

Comment:
you are supposed to download from upstream.

Reply:
I've obviously somewhat misunderstood how it is supposed to work.

The application is already being built by CI, I was trying not to duplicate that work and just have it packaged into a flatpak that could be loaded to flathub. I was referencing in part the OBS workflow: https://github.com/obsproject/obs-studio/blob/master/.github/workflows/flatpak.yml#L124

Is this feasible? I was able to find precious little about integrating flatpak into CI / CD.

Reply:
and how to you think flathub knows where to get it?

Reply:
OBS (also Firefox, few others) are special, but it requires too much handholding currently. It will be simpler if you could upload this tar.gz somewhere (be it S3, or GitHub releases), and reference it in your manifest.

Automating the process required to set up direct publishing is on the roadmap, but it's unlikely to happen within the next 6 months.

Reply:
Wasn't quite sure, though I was hoping something simple like: CI builds app, triggers flathub upload. :woman_shrugging: 

I think I've got it to a working place now though, thank you :+1: